src/table/components/TableWrapper.tsx

```diff
@@ -84,6 +87,10 @@ export default function TableWrapper(props: TableWrapperProps) {
     });
   }, [rows.length, totalRowCount, totalColumnCount, page]);
 
+  useDidUpdateEffect(() => {
+    setYScrollbarWidth(yScrollbarWidth);
+  }, [yScrollbarWidth]);
```

Comment:
`yScrollbarWidth` comes from the `useScrollbarWidth` hook and `setYScrollbarWidth` is actually a Table Context value.

But yes, it can still possible produce an infinite loop. Just not sure how it would could happen.